.storybook/preview.js

```diff
@@ -1,5 +1,4 @@
 import 'loki/configure-react';
-import { UiStyle } from '@catho/quantum-storybook-ui';
```

Comment:
What do you think to remove import on file `scripts/templates/component-story.js`?

Reply:
Its really should be removed...

Reply:
Its really should be removed...